lms/templates/video.html

```diff
@@ -23,7 +23,7 @@
           <div class="video-player-pre"></div>
           <section class="video-player">
               <div id="${id}"></div>
-              <h3 class="hidden">${_('No playable video sources found.')}</h3>
+              <h4 class="sr">${_('No playable video sources found.')}</h4>
```

Comment:
Are you sure this is supposed to be sr only?  I think this message is actually meant to hidden by default and then is shown when a video source isn't available.  By making it `.sr`, the screenreader would hear it when there is no real error, and a visual user wouldn't be presented the error message when there is an one.